Add Moodle section

ragusa87 opened the pull request:
#### Description

Add support for Moodle <https://moodle.org/>. This will read the version.php file on Moodle repository.
The icon is close to the Moodle logo.
#### Screenshot

![image](https://user-images.githubusercontent.com/1695207/57061414-06c98680-6cbd-11e9-9eed-7a0f0de8437b.png)

denysdovhan commented on sections/moodle.zsh:
```diff
@@ -0,0 +1,41 @@
+  # only if the version.php file contains $release
+  grep -q "\$release" version.php; [ $? -eq 0 ]  || return
+
+
```
Two blank lines.

denysdovhan commented on sections/moodle.zsh:
```diff
@@ -0,0 +1,41 @@
+
+
+
+  local moodle_version=$(cat version.php| grep -G "^\$release.*[0-9.+].*'" | awk '{print $3}' | awk '{print substr($1,2);}')
```
Add a blank line after this line.

ragusa87 commented:
@denysdovhan Hello, Thanks for the review, I corrected the whitespaces and added the missing documentation. I hope it's all right now, even if new section are frozen :stuck_out_tongue:. Have a nice day.

denysdovhan commented:
Hey, @ragusa87, Spaceship has changed the vector in terms of adding new section. This doesn't seem to fit requirements for new section.

Don't be upset, you can publish this as a custom section and add it to our registry!

ragusa87 commented:
Hello @denysdovhan, do you mean that I need to create a custom repository for this section instead of submitting a MR ?

In other word, a MR like the one below won't be merged ?
https://github.com/spaceship-prompt/spaceship-prompt/compare/master...ragusa87:feature/moodle?expand=1
